src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -172,23 +173,41 @@ const PastAnalysisTable = () => {
   }
 
   const updateRows = (listOfAnalysis: StudyInstanceWithSeries[]) => {
-    const rows: (tableRowsChild | tableRowsParent)[] = []
+    const rows: (tableRowsChild | tableRowsParent)[] = [];
     for (const analysis of listOfAnalysis) {
+      const validAnalyses = analysis.series.filter((series: ISeries) => series.classifications.size > 0);
+      const classifications = validAnalyses[0]?.classifications ? Array.from(validAnalyses[0]?.classifications?.keys()) : [];
```

Comment:
Could you please replace `Array.from()` with `spread` operator for the consistency purpose. I noticed we don't use `Array.from()` anywhere but do consistently use [...var] and {...var}, so it might be better to use the `spread` operator.
 
```suggestion
      const classifications = validAnalyses[0]?.classifications ? [...validAnalyses[0]?.classifications?.keys()] : [];
```

Reply:
@boris-nguyen, we actually tried using array spread initially, however it gave us `Type 'IterableIterator<string>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.ts(2569)`.

It seems like if we want to take advantage of this syntax, we'll need to update `tsconfig` with the `--downlevelIteration` flag. Is this something we should be doing?

Reply:
Can confirm that we did experience issues when trying to spread the keys of the map into an array. Not sure if there is a work-around but there's probably a better solution to be found; maybe we can take a look at this together later?